src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1004,5 +1004,37 @@ BOOST_AUTO_TEST_CASE(SimpleMeanSquaredErrorLayerTest)
   BOOST_REQUIRE_EQUAL(output.n_elem, 1);
 }
 
+/*
+ * Simple test for the Resize layer
+ */
+BOOST_AUTO_TEST_CASE(SimpleResizeLayerTest)
+{
+  arma::mat input, output, unzoomedOutput, expectedOutput;
+  size_t inRowSize = 10;
+  size_t inColSize = 10;
+  size_t outRowSize = 20;
+  size_t outColSize = 20;
+  input.set_size(inRowSize * inColSize, 1);
```

Comment:
I think we should add the `depth` parameter to the test. We didn't test the third dimension yet.